src/main/java/edu/ucla/library/avpairtree/verticles/MainVerticle.java

```diff
@@ -16,6 +18,8 @@
 import edu.ucla.library.avpairtree.MessageCodes;
 import edu.ucla.library.avpairtree.Op;
 import edu.ucla.library.avpairtree.handlers.StatusHandler;
+import edu.ucla.library.avpairtree.handlers.AmazonS3WaveformConsumer;
+import edu.ucla.library.avpairtree.handlers.LocalstackS3WaveformConsumer;
```

Comment:
I'm confused about the need for both of these. Is LocalstackS3WaveformConsumer just used for testing? In that case it seems like testing code is bleeding over into main code here. If so, I think it would be better to keep testing code confined to the test sources.

Reply:
Yes, just used for testing. I tried putting it with the test sources, but IIRC there was a resolution conflict of some kind. That was a while ago though, so I'll try again.